test/src/test_export_enum.rs

```diff
@@ -0,0 +1,30 @@
+use gdnative::prelude::*;
+
+#[derive(Debug, PartialEq, Clone, Copy, Export, ToVariant, FromVariant)]
+#[variant(enum = "repr")]
+#[repr(i32)]
+enum Dir {
+    Up = 1,
+    Down = -1,
+}
+
+pub(crate) fn run_tests() -> bool {
+    let mut ok = true;
+
+    ok &= test_from_variant();
+    ok &= test_to_variant();
+
+    ok
+}
+
+crate::godot_itest!(test_from_variant {
+    assert_eq!(Dir::from_variant(&1_i32.to_variant()), Ok(Dir::Up));
+    assert_eq!(Dir::from_variant(&(-1_i32).to_variant()), Ok(Dir::Down));
+    // 42 isn't mapped to any variant of `Dir`
+    assert!(Dir::from_variant(&42_i32.to_variant()).is_err());
+});
+
+crate::godot_itest!(test_to_variant {
+    assert_eq!(Dir::Up.to_variant(), 1_i32.to_variant());
+    assert_eq!(Dir::Down.to_variant(), (-1_i32).to_variant());
+});
```

Comment:
These tests are unnecessary. A more comprehensive version already exists at https://github.com/godot-rust/gdnative/blob/6dc972190f6cefb15eef6588c81a015983718df5/test/src/test_derive.rs#L173

Reply:
Got it. I will remove them. Do you think there is any other testcases should be added?  
Whether Godot rendering the property correctly seems to be a hard thing to check, I have no idea how to do that in integration tests.

Reply:
Yeah, I don't think that's particularly easy to automatically test. Technically I guess you can inspect the inspector from the editor tests, but that's going to be very fragile given the scene structure of the editor is I assume considered an implementation detail from Godot's perspective. I think the tests are fine as is.